src/components/Header/styles.module.scss

```diff
@@ -0,0 +1,190 @@
+@import 'base/variables';
+@import 'base/breakpoints';
+
+.navbar {
+  z-index: 200;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  background-color: $nav-color;
+}
+
+.nav-belt {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  min-height: 8vh;
+  width: 100%;
+  max-width: $max-container-width;
+  margin: 0 auto;
+  padding: 0 1em;
+}
+
+.logo {
+  color: $nav-labels-color;
+  letter-spacing: 0.2rem;
+  font-size: 1.5rem;
+  text-transform: uppercase;
+  text-decoration: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo-image {
+  width: 2rem;
+  height: 1.25rem;
+  background: url('../../assets/logo.svg') no-repeat center;
+  background-size: 100% 100%;
+}
+
+.nav-links {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  max-height: 50vh;
+  margin: 0;
+  width: 40%;
+
+  a {
+    text-decoration: none;
+    letter-spacing: 0.1rem;
+    font-size: 0.9rem;
+    font-weight: 700;
+    color: $nav-labels-color;
+  }
+
+  li {
+    list-style: none;
+  }
+}
+
+.nav-item-bold {
+  border: solid;
+  border-color: $main-color;
+  border-radius: 2rem;
+  padding: 0.25em 0.75em;
+
+  a {
+    color: $main-color;
+  }
+
+  &:hover {
+    background-color: $main-color;
+
+    a {
+      color: $background-color;
+    }
+  }
+}
+
+.burger {
+  display: none;
+}
+
+.burger div {
+  width: 1.25rem;
+  height: 0.15rem;
+  background-color: $nav-labels-color;
+  margin: 0.3em;
+}
+
+@include breakpoint('phone-md') {
```

Comment:
the menu is broken on mobile. 
![image](https://user-images.githubusercontent.com/21252506/88574618-370aec80-d008-11ea-8b16-be98ee359fae.png)

Reply:
That's how it looks in the template: https://gatsby-landing-page-starter.netlify.app/